### PR TITLE
moveTree - api conversion

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java [hunk 1]
```diff
@@ -64,22 +65,18 @@ protected String movePage(boolean moveUp) {
             }
             IPage currentPage = this.getPageManager().getDraftPage(selectedNode);
             if (!isUserAllowed(currentPage.getParent())) {
-                _logger.info("User not allowed!");
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java]: what happened to the loggers?

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java [hunk 2]
```diff
@@ -64,22 +65,18 @@ protected String movePage(boolean moveUp) {
             }
             IPage currentPage = this.getPageManager().getDraftPage(selectedNode);
             if (!isUserAllowed(currentPage.getParent())) {
-                _logger.info("User not allowed!");
                 this.addActionError(this.getText("error.page.userNotAllowed"));
                 return SUCCESS;
             }
             if (this.getPageManager().getRoot().getCode().equals(currentPage.getCode())) {
-                _logger.info("Root page cannot be moved");
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java]: this one is gone too

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java [hunk 3]
```diff
@@ -64,22 +65,18 @@ protected String movePage(boolean moveUp) {
...
+                this.addActionError(this.getText("error.page.movementHome.notAllowed"));
                 return SUCCESS;
             }
             boolean result = this.getPageManager().movePage(selectedNode, moveUp);
             if (!result) {
                 if (moveUp) {
-                    _logger.info("page movement not allowed");
                     this.addActionError(this.getText("error.page.movementUp.notAllowed"));
                 } else {
-                    _logger.info("page movement not allowed");
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java]: and this one as well, please check the whole file

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/page.xml [hunk 4]
```diff
@@ -156,37 +156,19 @@
...
-                <param name="root">lastUpdatePagesResponse</param>
-            </result>
-            <result name="failure" type="json">
-                <param name="root">lastUpdatePagesResponse</param>
-            </result>
-            <interceptor-ref name="jsonStack" />
-            <interceptor-ref name="entandoDefaultStack">
-                <param name="requestAuth.requiredPermission">enterBackend</param>
-            </interceptor-ref>
-        </action>
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/page.xml]: why is this one gone again?

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/page.xml [hunk 5]
```diff
@@ -156,37 +156,19 @@
             </interceptor-ref>
         </action>
 
-        <action name="status" class="pageFinderAction">
-            <result type="json">
-                <param name="root">pagesStatusResponse</param>
-            </result>
-            <result name="input" type="json">
-                <param name="root">pagesStatusResponse</param>
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/page.xml]: this action is gone too?

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java (hunk 1 above)
Comment (paddeo) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/PageTreeAction.java]: fixed loggers and config

**Comment by nicpuddu** [comment]
:shipit:

**Pull request merged**